### DISCUSSION TITLE
Help me change background back to it's original.. please!

### DISCUSSION THREAD

**Discussion by rizkyzhainuri** [Monika-After-Story/MonikaModDev#9855] [general]
I love the dark spaceroom it fits perfectly with the ambient music, please help me how to change back the background

**Comment by Wingdinggaster656** [comment]
If you are talking about the space room from original DDLC, then this job is a tough one.

MAS have changed the logic of some screen displays, and many of the current things can not perfectly adapt to the original speceroom. This is not an easy matter to solve. Maybe at some point in the future (and it is probably not a near point), MAS will add it back, but at present, this is simply impossible.
